### ISSUE TITLE
balance is incrementing

### ISSUE THREAD

**Issue by furesoft** [danielgerlag/NBlockchain#17]
when i type in sample currency balance after mine-genesis it adds 50 to the balance, every time i call balance command

**Comment by danielgerlag** [comment]
That's correct, that's what it is supposed to do. The first few blocks will be short and should adjust themselves to be longer after a few minutes
